src/components/biblio/BiblioEntity.js

```diff
@@ -174,29 +223,116 @@ const EntityTable = () => {
       <div>
         <LoadingOverlay active={isLoadingData || isLoadingMappings} />
         <Table bordered size="sm" responsive>
-          <thead>
-          <tr>
-            { headers.map( (header, index) => {
-              return (
-                  <th key={`tetTableHeader th ${index}`} >{header} {
-                    headersWithSortability.has(header) ?
-                        <FontAwesomeIcon icon={sortBy !== header || !descSort ? faSortAlphaDown : faSortAlphaUp} style={{color: sortBy === "entity_type" ? '#0069d9' : 'black'}}
-                                         onClick={() => {
-                                           if (sortBy === header && descSort) {
-                                             setSortBy(null);
-                                             setDescSort(true);
-                                           } else {
-                                             setSortBy(header);
-                                             setDescSort(!descSort)}
-                                         }}/> : null}</th>
-              )
-            } ) }
-            { source_headers.map( (header, index) => { return (<th key={`tetTableHeaderSource th ${index}`} >{header.startsWith('source_') ? header : 'source_' + header}</th>) } ) }
-          </tr>
+	  <thead>
```

Comment:
Can we create a separate component for the filter? I'm sure this will be reused for other columns.

Reply:
@valearna yes, we should. if not, the code will become so complicated if we are adding filtering for other columns. LOL

Reply:
again, it will be nice to work on it together in a pair/team programming session. @valearna

Reply:
Sounds good. Let's merge these changes and then we can work in pair programming to refactor the code